code/components/citizen-scripting-core/src/ResourceScriptingComponent.cpp

```diff
@@ -64,6 +64,8 @@ ResourceScriptingComponent::ResourceScriptingComponent(Resource* resource)
 #endif
 			);
 
+			auto sharedScripts = metaData->GetEntries("shared_script")

```

Comment:
Does this even compile? There's a _missing semicolon_.

Reply:
Yes, it compiles. But idk how it came out with a semicolon